course/page/code.py

```diff
@@ -908,18 +907,20 @@ def filter_img_attributes(tag, name, value):
                     if name in ["alt", "title"]:
                         return True
                     elif name == "src":
-                        return is_allowed_data_uri([
-                            "image/png",
-                            "image/jpeg",
-                            ], value)
+                        if is_allowed_data_uri([
+                                "image/png",
+                                "image/jpeg",
+                                ], value):
+                            return bleach.sanitizer.VALUE_SAFE
```

Comment:
Fix all <img> sanitized.

Reply:
We should migrate off of the bleach fork I created. This would be a good opportunity to do so. See #435.

This change looks good otherwise.

Reply:
OIC, can you push some change for that?

Reply:
Tested, adding `protocols=["data"]` will do the work. I'll also make sure `<script src="data:..."></script>` will be sanitized.